apache/incubator-zeppelin
# Add _site dir to .gitignore file

### What is this PR for?
When we execute Jekyll to build Zeppelin docs, `_site` directory is generated automatically. So I added `_site` to `.gitignore` file.

### What type of PR is it?
Improvement

### Todos
* [x] - Add `_site` dir to `.gitignore` file

### What is the Jira issue?
No

### How should this be tested?
1. Execute Jekyll following as in [here](https://github.com/apache/incubator-zeppelin/blob/master/docs/README.md)
2. Before adding `_site` to `.gitignore`, you can see that `git` is tracking `_site` directory. 
3. After applying this PR, `git` doesn't track this directory anymore. 

### Screenshots (if appropriate)

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No
